micrometer-test/src/main/java/io/micrometer/core/util/internal/logging/InternalMockLogger.java

```diff
@@ -0,0 +1,277 @@
+/**
+ * Copyright 2021 VMware, Inc.
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * https://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.micrometer.core.util.internal.logging;
```

Comment:
Is `internal` a good place for these (they are in the same package as the interfaces they implement)?

Reply:
Sounds like a good place to me.